### ISSUE TITLE
Ignore comments

### ISSUE THREAD

**Issue by gpyshenko** [oozcitak/xmlbuilder2#35]
It would be good to add a parameter for ignoring comments in xml.

**Comment by oozcitak** [comment]
Those adjustments will be possible when custom parsers and serializers are implemented. For now you can workaround by removing comment nodes after parsing, e.g.:
```js
const { create } = require('xmlbuilder2');

const doc = create('<root><!--comment1--><foo/><!--comment2--><bar/></root>');
doc.each(n => { if (n.node.nodeType === 8) n.remove(); }, false, true);
```

**Comment by gpyshenko** [comment]
Thank you. I understood.

**Comment by oozcitak** [comment]
I just released `v2.3.0` with custom parsers. Following example is from the [documentation](https://oozcitak.github.io/xmlbuilder2/custom-parsers.html):
```js
const { create } = require('xmlbuilder2');

const xmlString = `
<?xml version="1.0"?>
<!-- 3 records read from database -->
<records>
  <!-- node 1 found -->
  <record id="1"/>
  <!-- node 2 found -->
  <record id="2"/>
  <!-- node 3 not found -->
  <!-- node 4 found -->
  <record id="4"/>
</records>`;

// custom parser function returns undefined to skip comment nodes
const doc = create({ parser: { comment: () => undefined } }, xmlString);
console.log(doc.end({ prettyPrint: true }));
```
```xml
<?xml version="1.0"?>
<records>
  <record id="1"/>
  <record id="2"/>
  <record id="4"/>
</records>
```
